CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The changelog is available [on GitHub][2].
 
 ## Unreleased: 0.6.0.0
 
+* [#193](https://github.com/kowainik/relude/pull/193):
+  Implement `andM` et al. in terms of `&&^`.
```

Comment:
I would prefer if all affected functions are listed here, cause it is nicer to show the real changes and it could ease the life of users generally speaking :slightly_smiling_face:

Reply:
Done.